lib/ansible/modules/web_infrastructure/pmp.py

```diff
@@ -0,0 +1,296 @@
+  port:
+    required: false
+    default:  7272
+    description:
+      - Base URI for the PMP instance.
+  operation:
+    required: true
+    aliases: [ command ]
+    choices: [ create_host ]
+    description:
+      - The operation to perform.
+
+  token:
+    required: true
+    description:
+      - Token of the API user that performs the action
+
+  group:
+    required: false
+    description:
+      - If exists set the main group of the host.
+
+  resource_name:
+    required: true
+    description:
+     - The name of the resource that will be created.
+
+  account_name:
+    required: true
+    description:
+     - User associated with the created resource.
+
+  password:
+    required: true
+    description:
+     - Password asociated with the user_name.
+
+  owner:
+    required: true
+    description:
+     - Change the default owner, must be an exising user.
+
+  use_proxy:
+    required: false
+    description:
+      - Set if open_url must use proxy
+  validate_certs:
+    required: false
+    description:
+      - Set if open_url must validate certificates
+
+author: "Bernat Mut (@berni69)"
+"""
+
+EXAMPLES = """
+# Retrieve a password:
+- name: Retrieve Password
+  pmp:
+    uri: '{{ pmp_server }}'
+    port: '{{ pmp_port }}'
+    token: '{{ user_token }}'
+    command: 'get_password'
+    resource_name: '{{ server_name }}'
+    account_name: '{{ user_name }}'
+  register: issue
+
+- name: Create server host
+  pmp:
+    uri: '{{ pmp_server }}'
+    port: '{{ pmp_port }}'
+    token: '{{ user_token }}'
+    resource_name: '{{ server_name }}'
+    group: '{{ my_group}}'
+    account_name: '{{ user_name }}'
+    password: '{{ password }}'
+    owner: 'DOMAIN\\USER'
+
+"""
+
+RETURN = '''
+status:
+    description: status of action executed
+    returned: success
+    type: string
+    sample: 'OK'
+password:
+    description: password retrieved
+    returned: success
+    type: string
+    sample: "123456"
+'''
+
+import json
+import sys
+import urllib
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.urls import open_url
+
+
+class PasswordManagerPro:
+    data = {
+        'HOST': '',
+        'PORT': '',
+        'TOKEN': '',
+    }
+
+    use_proxy = False
+    validate_certs = False
+
+    def format(self, strr):
+        for key, value in self.data.items():
+            strr = strr.replace('{' + key + '}', value)
+        return strr
+
+    def __init__(self, args=None):
+        if args:
+            if 'TOKEN' in args:
+                self.data['TOKEN'] = args['TOKEN']
+            if 'HOST' in args:
+                self.data['HOST'] = args['HOST']
+            if 'HOST' in args:
+                self.data['PORT'] = args['PORT']
+            if 'use_proxy' in args:
+                self.use_proxy = args['use_proxy']
+            if 'validate_certs' in args:
+                self.validate_certs = args['validate_certs']
+
+    def getResources(self):
```

Comment:
This is just style but we try to follow standard python naming conventions.  So function and method names should be underscore separated like this: ```get_resources```.